br/tests/br_full_cluster_restore/run.sh

```
@@ -27,6 +28,11 @@ restart_services
 
 unset BR_LOG_TO_TERM
 run_sql_file $CUR/full_data.sql
+run_sql "select * from mysql.global_priv;"
+run_sql_as user3 "123456" "show status like 'Ssl_cipher';"
+cat $TEST_DIR/sql_res.$TEST_NAME.txt
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)